core/indexer/elasticProcessor.go

```diff
@@ -430,6 +480,86 @@ func (ei *elasticProcessor) SaveRoundsInfo(infos []workItems.RoundInfo) error {
 	return ei.elasticClient.DoBulkRequest(&buff, roundIndex)
 }
 
+func (ei *elasticProcessor) indexAlteredAccounts(accounts map[string]struct{}) {
+	for address := range accounts {
+		addressBytes, err := ei.addressPubkeyConverter.Decode(address)
+		if err != nil {
+			log.Warn("cannot decode address", "address", address, "error", err)
+			continue
+		}
+
+		account, err := ei.accountsDB.LoadAccount(addressBytes)
+		if err != nil {
+			log.Warn("cannot load account", "address bytes", addressBytes, "error", err)
+			continue
+		}
+
+		userAccount, ok := account.(state.UserAccountHandler)
+		if !ok {
+			log.Warn("cannot cast AccountHandler to type UserAccountHandler")
+			continue
+		}
+
+		err = ei.SaveAccount(userAccount)
+		if err != nil {
+			log.Warn("cannot index account",
+				"address", ei.addressPubkeyConverter.Encode(userAccount.AddressBytes()),
+				"error", err)
+		}
+	}
+}
+
+// SaveAccount will prepare and save information about an account in elasticsearch server
+func (ei *elasticProcessor) SaveAccount(account state.UserAccountHandler) error {
+	if _, ok := ei.enabledIndexes[accountsIndex]; !ok {
+		return nil
+	}
+
+	if check.IfNil(account) {
+		return nil
+	}
+
+	var buff bytes.Buffer
+
+	balanceAsFloat := ei.computeBalanceAsFloat(account.GetBalance())
+	acc := AccountInfo{
+		Address:    ei.addressPubkeyConverter.Encode(account.AddressBytes()),
+		Nonce:      account.GetNonce(),
+		Balance:    account.GetBalance().String(),
+		BalanceNum: balanceAsFloat,
+	}
+
+	accBytes, err := json.Marshal(&acc)
+	if err != nil {
+		return err
+	}
+
+	buff.Grow(len(accBytes))
+	_, err = buff.Write(accBytes)
+	if err != nil {
+		log.Warn("elastic search: save account, write", "error", err.Error())
+	}
+
+	req := &esapi.IndexRequest{
+		Index:      accountsIndex,
+		DocumentID: ei.addressPubkeyConverter.Encode(account.AddressBytes()),
+		Body:       bytes.NewReader(buff.Bytes()),
+		Refresh:    "true",
+	}
+
+	return ei.elasticClient.DoRequest(req)
```

Comment:
instead of DoRequest you have to use DoBulkRequest to save more than one account per request. You should create a request with size smaller than 1MB and send it.

Reply:
done